.circleci/config.yml

```diff
@@ -116,4 +116,4 @@ jobs:
             sudo mv trivy /usr/local/bin
       - run:
           name: Scan the local image with trivy
-          command: trivy --exit-code 0 --no-progress trivy-ci-test:${CIRCLE_SHA1}
+          command: trivy --exit-code 1 --severity HIGH --no-progress trivy-ci-test:${CIRCLE_SHA1}
```

Comment:
@quoideneuf In looking at this, I wonder if both high and critical need to be stated. They are listing both medium and high in the first example.
$ trivy image --exit-code 0 --severity MEDIUM,HIGH ruby:2.4.0
$ trivy image --exit-code 1 --severity CRITICAL ruby:2.4.0

Reply:
@sensei100 I believe you are right. Fixed now.